Vagrantfile

```diff
@@ -132,10 +132,13 @@ apiServer:
   host: #{IPAddr.new(CONTROL_PLANE_IP).mask(CONTROL_PLANE_NETMASK).to_range.last(2).first.to_s}
   keepalived:
     enabled: true
+products:
+  metalk8s:
+  - /srv/scality/metalk8s-$VERSION
```

Comment:
No that's not what we want it's the oposite.

The path where to extract the iso should be hardcoded
We should have something like
```
products:
  metalk8s:
    - /root/metalk8s.iso
    - /var/tmp/metalk8s.iso
    - /toto/tata/titi.iso
```
And then the goal is to mount these iso in function of the version so for each `/srv/scality/metalk8s-<version>`

Reply:
Yes, it's already the case, the path where to mount the iso is hardcoded in `/srv/scality`, it is just that both are supported: product path and iso (if the iso is given, mount it otherwise, just config salt and perform a set of assert)